dev/sparktestsupport/modules.py

```diff
@@ -481,7 +481,7 @@ def __hash__(self):
 
 sparkr = Module(
     name="sparkr",
-    dependencies=[hive, mllib],
+    dependencies=[hive, mllib, hive_thriftserver],
```

Comment:
Since SparkR already depends on hive, this PR just adds the dependence on `hive_thriftserver`